gen/ad_mu

```diff
@@ -231,18 +208,16 @@ sub processGroups {
 			my $key = "CN=".$adName."_group.muni.cz,OU=".$ouName.",".$baseDNGroups;
 
 			$groups->{$ouName}->{$key}->{$A_G_AD_NAME} = $adName;
+			my $displayName = $data->getGroupAttributeValue( group => $groupId, attrName => $A_G_AD_DISPLAY_NAME );
```

Comment:
This line of code already exists above.

Reply:
I missed this one. Probably forgoten line of code on the test instance. I have removed it.